server/models/userModel.js

```diff
@@ -0,0 +1,81 @@
+import Data from '../db/db';
+
+class UsersModel {
+	constructor(payload = null) {
+		this.payload = payload;
+		this.result = null;
+	}
+
+	async createUser() {
+		const user = {
+			firstname: this.payload.firstName,
```

Comment:
use object destructuring to get the fields from the payload, it would simplify your work